.github/workflows/test_examples.yml

```diff
@@ -5,11 +5,13 @@ on:
       - 'tensorflow/**'
       - 'examples/*/*.robot'
       - 'examples/*/build.sh'
+      - '.github/workflows/test_examples.yml'
```

Comment:
is there any reason to have these filters fot this job?

Reply:
I mean - I'd rather not have it at all if we can. We might filter out ipynb files